src/components/movie/AddMovie.vue

```diff
@@ -0,0 +1,176 @@
+<script lang="ts" setup>
+useHead({
+  title: () => "Add Movie - Basement of Stinkology",
+});
+
+const search = ref("");
+const selectedMovie = ref<Movie | null>(null);
+
+const { data } = await useFetch(`/api/movies/search`, {
+  query: {
+    search,
+  },
+  watch: [search],
+});
+
+function handleSearchUpdateValue(value: string) {
+  search.value = value;
+}
+
+interface Movie {
+  adult?: boolean;
+  backdrop_path?: string;
+  genre_ids?: Array<number>;
+  id?: number;
+  original_language?: string;
+  original_title?: string;
+  overview?: string;
+  popularity?: number;
+  poster_path?: string;
+  release_date?: string;
+  title?: string;
+  video?: boolean;
+  vote_average?: number;
+  vote_count?: number;
+}
+
+function handleMovieClick(movie: Movie) {
+  selectedMovie.value = movie;
+}
+
+const isSubmitting = ref(false);
+
+const input = ref();
+useFocus(input, { initialValue: true });
+const { path } = useRoute();
+
+const thePath = path.replace("/add-movie", "");
```

Comment:
```suggestion
const parentPath = path.replace("/add-movie", "");
```

Would `parentPath` be more descriptive here? Or `prevPath`, or something similar?